application/controllers/home.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * 
+ * home is a controller
+ * 
+ */
+
+class home
+{
+  /**
+   * 
+   * index is a function which checks if login page is present in the views
+   * folder and if it is present then the page is embedded
+   * 
+   */
+
+  public function index() {
+    if (file_exists('application/views/login.php')) {
+      require_once 'application/views/login.php';
+    }
+  }
+
+  /**
+   * 
+   * viewRegister is a function which checks if register page is present in the 
+   * views folder and if it is present then the page is embedded
+   * 
+   */
```

Comment:
As per coding standards, the first letter should be capital and the line should end with (.) and also mention the return type of the function.